forge-gui/res/cardsfolder/upcoming/urborg_repossession.txt

```diff
@@ -0,0 +1,10 @@
+Name:Urborg Repossession
+ManaCost:B
+Types:Sorcery
+K:Kicker:1 G
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Hand | ChangeNum$ 1 | TgtPrompt$ Choose target creature card in your graveyard | ChangeType$ Creature.YouOwn | SubAbility$ DBGainLife | SpellDescription$ Return target creature card from your graveyard to your hand. You gain 2 life. If this spell was kicked, return another target permanent card from your graveyard to your hand.
+SVar:DBGainLife:DB$ GainLife | LifeAmount$ 2| SubAbility$ DBReturnKicked
+SVar:DBReturnKicked:DB$ ChangeZone | Condition$ Kicked | TargetUnique$ True | Origin$ Graveyard | Destination$ Hand | ChangeNum$ 1 | TgtPrompt$ Choose target permanent card in your graveyard | ChangeType$ Permanent.YouOwn 
```

Comment:
This and line 5 need to target with ValidTgts$

Right now you are mixing the targeting params with non-targeting params